web/src/app/administration/components/access-items-management/access-items-management.component.ts

```diff
@@ -83,14 +81,7 @@ export class AccessItemsManagementComponent implements OnInit {
   searchForAccessItemsWorkbaskets() {
     this.removeFocus();
     this.store
-      .dispatch(
-        new GetAccessItems(
-          [this.accessId, ...this.groups],
-          this.accessItemsForm ? this.accessItemsForm.value.accessIdFilter : undefined,
-          this.accessItemsForm ? this.accessItemsForm.value.workbasketKeyFilter : undefined,
-          this.sortModel
-        )
-      )
+      .dispatch(new GetAccessItems([this.accessId, ...this.groups], '', '', this.sortModel))
```

Comment:
Now, the action GetAccessItems contains logic which is never used (the filter). In case we are sure that we don't need this again, the action GetAccessItems could be modified. We could also create a new ticket and modify this after the redesign.